src/pymor/operators/numpy.py

```diff
@@ -319,7 +319,12 @@ def apply_inverse(self, V, mu=None, initial_guess=None, least_squares=False,
         else:
             if least_squares:
                 try:
-                    R, _, _, _ = np.linalg.lstsq(self.matrix, V.to_numpy().T)
+                    import pymor.config
+                    if tuple(int(v) for v in pymor.config.NUMPY_VERSION.split('.')) >= (1, 14, 0):
```

Comment:
```suggestion
                    from packaging.version import parse
                    if parse(pymor.config.NUMPY_VERSION) >= parse('1.14'):
```

Reply:
Ah, I knew we had a package for comparing versions, but I was to lazy too look up its name. Thanks ..